tutorials/composition/Interpolated Waveforms.ipynb

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
<p>"Naturally, the valid parameters will depend on the chosen interpolator."</p><p>Not sure what is meant by that. Is this a reference to the fact that Omega should not be negative and thus you should be cautious when choosing the interpolation method or is this relating to the extra args one can feed the interpolator ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>It's about the extra args, I guess it's not clear when written in this way. What if I change to:</p><p> </p><p>"Naturally, the choice of interpolator will dictate which extra parameters can be optionally provided."</p><p> </p><p> </p>

Reply:
Yes, it's clearer